doc/TUTORIAL.md

```diff
@@ -96,32 +93,35 @@ Each service has a directory for its files:
 All service badge classes inherit from [BaseService] or another class which extends it.
 Other classes implement useful behavior on top of [BaseService].
 
-* [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
+- [BaseJsonService](https://github.com/badges/shields/blob/master/services/base-json.js)
   implements methods for performing requests to a JSON API and schema validation.
-* [BaseXmlService](https://github.com/badges/shields/blob/master/services/base-xml.js)
+- [BaseXmlService](https://github.com/badges/shields/blob/master/services/base-xml.js)
   implements methods for performing requests to an XML API and schema validation.
-* If you are contributing to a *service family*, you may define a common super
+- If you are contributing to a _service family_, you may define a common super
   class for the badges or one may already exist.
 
-[BaseService]: https://github.com/badges/shields/blob/master/services/base.js
+[baseservice]: https://github.com/badges/shields/blob/master/services/base.js
 
 As a first step we will look at the code for an example which generates a badge without contacting an API.
 
 ```js
-'use strict'                                         // (1)
+'use strict' // (1)
 
-const { BaseService } = require('..')                // (2)
+const { BaseService } = require('..') // (2)
 
-module.exports = class Example extends BaseService { // (3)
+module.exports = class Example extends BaseService {
+  // (3)
```

Comment:
I'm still not a fan of how prettier moves these comments around (especially num. 3) due to their relation to the bullets below. 

Maybe we could move all the numbered comments to above the line the reference to both be consistent and satisfy prettier?

This doesn't need to be a blocker IMO (given all the changes in this PR) but if we merge these Tutorial changes as-is then I do feel we should update these inline code blocks soon

Reply:
Ahhh my editor must have done that. Thanks for catching it.

Do you think the inline numbers would look okay? They would be a _lot_ easier to maintain, and it'd let us stop prettier-ignoring this file.

Reply:
How about we merge this and one of us can deal with the broken links and formatting as a follow-on?

Reply:
> How about we merge this and one of us can deal with the broken links and formatting as a follow-on?

Works for me!

Reply:
I opened https://github.com/badges/shields/issues/3116 just to make sure we don't accidentally forget